assets/js/timetree.js

```diff
@@ -435,32 +421,62 @@ function TreeGraph({ nodes, links, centuries }) {
     .attr("dy", LeafLabel.lineHeight); // delta-y : relative position based on line height
 
   // visual debugging for layout
-
-  // debug label placement + collision radius
-  //  draw a circle around each label with calculated radius
+  // draw circles and lines in a debug layer that can be shown or hidden
+  // circle size for leaf and leaf label matches radius used for collision
+  // avoidance in the network layout
   debugLayer
-    .selectAll("circle.debug-label")
-    .data(nodes.filter((d) => d.type == "leaf-label"))
+    .selectAll("circle.debug")
+    .data(nodes) // .filter((d) => d.type == "leaf-label"))
     .join("circle")
-    .attr("class", "debug-label")
+    .attr(
+      "class",
+      (d) => `debug debug-${d.type} dbg-${getBranchStyle(d.branch) || ""}`
+    )
     .attr("cx", (d) => d.x)
     .attr("cy", (d) => d.y)
-    .attr("r", (d) => d.radius)
-    .attr("stroke", "yellow")
-    .attr("fill", "transparent");
+    .attr("r", (d) => {
+      // NOTE: we're currently adjusting collision radius slightly
+      // in the network simulation; probably needs some
+      // adjusments, and should make sure these match
+      if (d.type == "leaf-label") {
+        return leafSize.width - 5;
+        // return d.label.radius;
+      }
+      if (d.type == "leaf") {
+        return leafSize.width - 5;
+        // return leafSize.width;
+      }
+      // note: this is larger than collision radius, increase size for visibility
+      return 5; // for branch nodes
```

Comment:
This logic is only for the debug view?

Reply:
I was previously displaying circles with the exact width and label radius, but then recognized it didn't match the logic I'm using for collision avoidance in the d3 simulation right now (this is why labels overlap some). I meant to adjust it to match, but I see here that I mistakenly copied the same thing twice! Let me push a fix for that.